Simplenote/src/main/res/layout/activity_collaborators.xml

```diff
@@ -1,95 +1,56 @@
-                android:lineSpacingExtra="6sp"
-                android:text="@string/shared_with"
-                android:textColor="?attr/colorAccent"
-                android:textSize="14sp"
-                android:textStyle="normal"
-                android:visibility="gone"
-                tools:visibility="visible"/>
-
-            <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/collaborators_list"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="72dp"
-                android:layout_marginBottom="10dp"
-                android:visibility="gone"
-                tools:visibility="visible"/>
-
-            <View
-                android:id="@+id/divider_line"
-                android:layout_width="match_parent"
-                android:layout_height="1dp"
-                android:background="?attr/dividerColor"
-                android:visibility="gone"
-                tools:visibility="visible"/>
-
-        <LinearLayout
-            android:id="@+id/row_add_collaborator"
+        <com.automattic.simplenote.widgets.EmptyViewRecyclerView
+            android:id="@+id/collaborators_list"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="?android:attr/selectableItemBackground"
-            android:orientation="vertical">
-            <TextView
-                android:id="@+id/button_add_collaborator"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="72dp"
-                android:layout_marginTop="23dp"
-                android:layout_marginBottom="23dp"
-                android:text="@string/add_collaborator"
-                android:textAllCaps="false"
-                android:textAppearance="?android:attr/textAppearanceMedium"
-                android:textColor="?attr/colorAccent"
-                android:visibility="gone"
-                tools:visibility="visible"/>
-        </LinearLayout>
-
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical">
+            android:layout_height="match_parent"
+            android:layout_marginStart="72dp"
+            android:layout_marginBottom="10dp"
+            android:background="@android:color/transparent"
+            android:clipToPadding="false"
+            android:divider="?attr/listDividerDrawable"
+            android:dividerHeight="@dimen/divider_height"
+            android:paddingBottom="@dimen/padding_list_bottom"
+            android:scrollbarStyle="outsideOverlay"
+            android:scrollbars="vertical" />
+
+        <include
+            android:id="@+id/empty"
+            layout="@layout/empty_view_message" />
+
+        <ImageButton
```

Comment:
This is something that I have not think of since I just want to keep the screens consistent with the other screens on the app. There are other changes in the layout files that also need some improvements such as naming convention, etc. Thus, let me make a list of things to improve in the views and add this case in it to see if it is feasible for us to change all the `FAB` buttons to the material one.

Reply:
Thanks for even thinking about that @danilo04 ! 🙏